lua/anywise_reg/handlers.lua

```diff
@@ -29,15 +29,15 @@ M.handle_paste_behind = function(prefix)
     cmd.normal(prefix .. 'P')
 end
 
-M.handle_paste = function(prefix)
+M.handle_paste = function(prefix, operator)
     local reg = get_register(prefix)
     local d = data.reg_data[reg]
     if d ~= nil then
         -- go to end of current text object
-        cmd.normal('v'..d.textobject, {noremap = false})
+        cmd.normal('v' .. d.textobject, {noremap = false})
```

Comment:
I guess there should be an if-statement here to check for `P` and if so do `o`

Reply:
Oh sorry I forgot about this, do you think it is possible there are other paste operators? that would need different handling of this section? so should we make it configurable?

Reply:
No problem, I think for now it's fine to assume that it's only `p` and `P`. If there will be others we can change that then :)